brion/target.cpp

```diff
@@ -171,7 +172,11 @@ void _parse( const Targets& targets, const std::string& name, GIDSet& gids )
 GIDSet Target::parse( const Targets& targets, const std::string& name )
 {
     brion::GIDSet gids;
-    _parse( targets, name, gids );
+    bool found(false);
+    _parse( targets, name, gids, found );
+    if( !found )
```

Comment:
isn't it easier to check for gids.empty() here which should lead to the same result? Why throw at all?